src/content/supported-icons.tsx

```diff
@@ -35,14 +35,22 @@ import { IconTerminal16 } from '@hashicorp/flight-icons/svg-react/terminal-16'
 import { IconTerminalScreen16 } from '@hashicorp/flight-icons/svg-react/terminal-screen-16'
 import { IconTerraformColor16 } from '@hashicorp/flight-icons/svg-react/terraform-color-16'
 import { IconTools16 } from '@hashicorp/flight-icons/svg-react/tools-16'
-import { IconVaultColor16 } from '@hashicorp/flight-icons/svg-react/vault-color-16'
+import { IconVault16 } from '@hashicorp/flight-icons/svg-react/vault-16'
 import { IconVmware16 } from '@hashicorp/flight-icons/svg-react/vmware-16'
 import { IconWaypointColor16 } from '@hashicorp/flight-icons/svg-react/waypoint-color-16'
 import { IconWrench16 } from '@hashicorp/flight-icons/svg-react/wrench-16'
 
+/**
+ * @TODO refactor to remove the HashiCorp product logos from this list
+ * when product logos are needed, component should use `ProductIcon`
+ *
+ * With the introduction of theming, when adding new icons to this list,
+ * consider both dark and light states.
+ */
+
 export const SUPPORTED_ICONS = {
 	api: <IconApi16 />,
-	'aws-color': <IconAwsColor16 />,
+	aws: <IconAws16 />,
```

Comment:
'aws' is the outlier here, I decided to remove it from rendering the color version to avoid making all of these supported icons support theming. Until we have a larger or more broad usecase, we will do this until we have a stance on external company icon styles in the app. External company logos will always be troublesome with theming, so I'm proposing that we consider always using just black or white for these icons. More discussion need to be had now.

Reply:
Additionally, this approach doesn't deviate from the existing patterns where generic icons use black or white fills. See example here: 

<img width="944" alt="Screenshot 2023-03-20 at 5 22 17 PM" src="https://user-images.githubusercontent.com/36613477/226493009-37788270-fddf-4288-b2a5-88a43d167c79.png">

Reply:
In the future, if the flight library's aws icon allows the 'color' of the wordmark to be inherited by the parent, we could work around this (like how we handle terraform clouds logo for example). But given the legality of logo use, I can see why they wouldn't be able to do this.